test/tests.js

```diff
@@ -1053,6 +1070,30 @@ describe('pasync', function() {
 		}).catch(done);
 	});
 
+	it('memoize', function(done) {
+		var myFunc = function(up, down, no) {
+			return Promise.resolve(up * down * no);
+		};
+		var myMemo = pasync.memoize(myFunc);
+		myMemo(2, 4, 3).then(function(result) {
+			expect(result).to.equal(24);
+			done();
+		}).catch(done);
+	});
+
+	it('unmemoize', function(done) {
+		var myMemo;
+		var myFunc = function(up) {
+			return Promise.resolve(up * 2);
+		};
+		myMemo = pasync.memoize(myFunc);
+		var noMemo = pasync.unmemoize(myMemo);
+		noMemo(2).then(function(thing) {
+			expect(thing).to.equal(4);
```

Comment:
Nest this call, to ensure the memoize function is called all the time (since nothing should be memoized in unmemoize).